src/components/Menu/menu.css

```diff
@@ -0,0 +1,32 @@
+.menu-page {
+  margin: 0 auto;
+  padding: 0;
+}
+
+.menu-page li:not(:last-child) {
+  margin-bottom: 30px;
+}
+
+.menu-page li {
+  list-style-type: none;
+}
+
+.menu-links {
+  text-align: center;
+  padding: 10px 0;
+  background-color: var(--primary-element-color);
+  border-radius: 30px;
+  display: block;
+  text-decoration: none;
+  color: var(--secondary-text-color);
```

Comment:
Please, remind me explain about priority practices for css rules

Reply:
done